### PR TITLE
Add DragonPvP

### PR DESCRIPTION
**Pull request by SonoAX**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/dragonpvp/manifest.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "nice_name": "DragonPvP Network",
+    "direct_ip": "play.dragonpvp.it",
+    "supported_languages": [
+      "it"
+    ],
+    "social": {
+      "web": "https://www.dragonpvp.site",
+      "web_shop": "https://www.dragonpvp.site/store",
+      "web_support": "https://www.dragonpvp.site/forum",
+      "discord": "https://dsc.gg/dsdragonpvp"
```
Comment (JakobBruening) [inline minecraft_servers/dragonpvp/manifest.json:12]: Please use official discord urls.

File: minecraft_servers/dragonpvp/manifest.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      },
+      "uhc": {
+        "name": "UHC",
+        "color": "#537928",
+        "command": "/queue uhc"
+      }
+    },
+    "discord": {
+      "server_id": 884523075503259709,
+      "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/dragonpvp/manifest.json:34]: Please use false, as soon is only allowed for verified partners.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/dragonpvp/manifest.json (hunk 2 above)
Comment (SonoAX) [reply minecraft_servers/dragonpvp/manifest.json]: done

File: minecraft_servers/dragonpvp/manifest.json (hunk 1 above)
Comment (SonoAX) [reply minecraft_servers/dragonpvp/manifest.json]: done

**Pull request merged**